consensus/service/src/attested_api_service.rs

```diff
@@ -10,31 +10,41 @@ use mc_common::{
     HashSet,
 };
 use mc_consensus_enclave::ConsensusEnclave;
-use mc_util_grpc::{rpc_logger, rpc_permissions_error, send_result};
+use mc_util_grpc::{auth::Authenticator, rpc_logger, rpc_permissions_error, send_result};
 use mc_util_metrics::SVC_COUNTERS;
 use std::sync::{Arc, Mutex};
 
 #[derive(Clone)]
-pub struct AttestedApiService<E: ConsensusEnclave + Send, S: Session> {
-    enclave: E,
+pub struct AttestedApiService<S: Session> {
+    enclave: Arc<dyn ConsensusEnclave + Send + Sync>,
```

Comment:
The change to `Arc<dyn>` follows the direction we started taking for the `mockall` tests.